app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user, foreign_key: 'author_id'
+  belongs_to :post
+
+  def update_likes_counter(post)
+    post.likes_counter = Like.where(post_id: post.id).count
+  end
```

Comment:
[OPTIONAL]
- You could use [rails active callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) to call this method every time a new post is saved to the database. After that, you can make the method private. It can look like this
```
  after_save :update_likes_counter

  private

  def update_likes_counter(post)
    post.likes_counter = Like.where(post_id: post.id).count
  end
```